skills/cuopt-developer/resources/python_bindings.md

```
@@ -0,0 +1,233 @@
+|-------|------|---------|
+| Library loader | `python/libcuopt/libcuopt/load.py` | Dynamically loads `libcuopt.so` via ctypes |
+| Python API | `python/cuopt/cuopt/linear_programming/` | User-facing classes (`Problem`, `SolverSettings`) |
+| Python API | `python/cuopt/cuopt/routing/` | Routing API |
+| Cython bindings | `python/cuopt/cuopt/linear_programming/solver/solver_wrapper.pyx` | Solver bridge |
+| Cython bindings | `python/cuopt/cuopt/linear_programming/data_model/data_model_wrapper.pyx` | Data model bridge |
+| Cython declarations | `python/cuopt/cuopt/linear_programming/solver/solver.pxd` | C++ interface declarations |
+| Cython declarations | `python/cuopt/cuopt/linear_programming/data_model/data_model.pxd` | C++ interface declarations |
+| C++ headers | `cpp/include/cuopt/linear_programming/` | Public API |
+| C++ implementation | `cpp/src/` | Solver internals |
+
+## File Types
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)